lib/Cake/View/Errors/fatal_error.ctp

```diff
@@ -36,4 +36,3 @@
 if (extension_loaded('xdebug')) {
 	xdebug_print_function_stack();
 }
-?>
```

Comment:
It sees not, at least not for 2.x and not explicitly:

http://book.cakephp.org/2.0/en/contributing/cakephp-coding-conventions.html#view-files